test/tests/executioners/quadratic_minimize.i

```diff
@@ -6,7 +6,7 @@
   parameter_names = 'results'
   num_values = 3
   initial_condition = '5 8 1'
-  objective = 1.0
+  objective = 0.0
```

Comment:
Why this change?

Reply:
these tests are no longer relevant. We need to put in unit tests where we don't build the hessian but apply it.

Reply:
I"ll add an issue for this.